.github/workflows/e2e-test.yaml

```diff
@@ -142,7 +142,28 @@ jobs:
           username: ${{ secrets.NEON_DOCKERHUB_USERNAME }}
           password: ${{ secrets.NEON_DOCKERHUB_PASSWORD }}
 
+      # https://docs.k3s.io/installation/private-registry#registries-configuration-file
+      # https://github.com/neondatabase/autoscaling/issues/975
+      - name: set k3d registries.yaml
+        # TODO: Implement an equivalent for kind?
```

Comment:
Maybe just drop kind support?

Reply:
I still use kind locally because it says "have a nice day" :sweat_smile:

Given that k3d _is_ a little bit different from a more normal k8s distribution, I'd rather we keep kind around for now -- if only so we make sure that we don't implement something that works only in k3d (but not EKS).

WDYT?

Reply:
Sure thing!